packages/devextreme/js/ui/card_view.d.ts

```diff
@@ -0,0 +1,1880 @@
+    onCardClick?: (e: CardClickEvent) => void; // TODO: sync with impl
+    /**
+     * @docid
+     * @public
+     * @type_function_param1 e:{ui/card_view:CardDblClickEvent}
+     * @action
+     */
+    onCardDblClick?: (e: CardDblClickEvent) => void; // TODO: sync with impl
+    /**
+     * @docid
+     * @public
+     * @type_function_param1 e:{ui/card_view:CardPreparedEvent}
+     * @action
+     */
+    onCardPrepared?: (e: CardPreparedEvent) => void; // TODO: sync with impl
+    /**
+     * @docid
+     * @public
+     * @type_function_param1 e:{ui/card_view:FieldCaptionClickEvent}
+     * @action
+     */
+    onFieldCaptionClick?: (e: FieldCaptionClickEvent) => void; // TODO: sync with impl
+    /**
+     * @docid
+     * @public
+     * @type_function_param1 e:{ui/card_view:FieldCaptionDblClickEvent}
+     * @action
+     */
+    onFieldCaptionDblClick?: (e: FieldCaptionDblClickEvent) => void; // TODO: sync with impl
+    /**
+     * @docid
+     * @public
+     * @type_function_param1 e:{ui/card_view:FieldCaptionPreparedEvent}
+     * @action
+     */
+    onFieldCaptionPrepared?: (e: FieldCaptionPreparedEvent) => void; // TODO: sync with impl
+    /**
+     * @docid
+     * @public
+     * @type_function_param1 e:{ui/card_view:FieldValueClickEvent}
+     * @action
+     */
+    onFieldValueClick?: (e: FieldValueClickEvent) => void; // TODO: sync with impl
+    /**
+     * @docid
+     * @public
+     * @type_function_param1 e:{ui/card_view:FieldValueDblClickEvent}
+     * @action
+     */
+    onFieldValueDblClick?: (e: FieldValueDblClickEvent) => void; // TODO: sync with impl
+    /**
+     * @docid
+     * @public
+     * @type_function_param1 e:{ui/card_view:FieldValuePreparedEvent}
+     * @action
+     */
+    onFieldValuePrepared?: (e: FieldValuePreparedEvent) => void; // TODO: sync with impl
+    /**
+     * @docid
+     * @public
+     * @type_function_param1 e:{ui/card_view:CardHoverChangedEvent}
+     * @action
+     */
+    onCardHoverChanged?: (e: CardHoverChangedEvent) => void; // TODO: sync with impl
+    /**
+     * @docid
+     * @public
+     */
+    cardFooterTemplate?: template | ((data: CardTemplateData, container: DxElement) => string | UserDefinedElement);
+    /**
+     * @docid
+     * @public
+     */
+    cardHeader?: CardHeader;
+
+    /**
+     * @docid
+     * @public
+     */
+    hoverStateEnabled?: boolean; // TODO: sync with impl
+
+    // #endregion
+
+    // #region Toolbar
+
+    /**
+     * @docid
+     * @public
+     */
+    toolbar?: Toolbar;
+
+    // #endregion
+
+    sorting?: Sorting;
+
+    /**
+     * @docid
+     * @type Filter expression
+     * @default null
+     * @fires dxCardViewOptions.onOptionChanged
+     * @public
+     */
+    filterValue?: string | Array<any> | Function;
+    /**
+     * @docid
+     * @public
+     * @type object
+     */
+    filterBuilderPopup?: PopupProperties;
+    /**
+     * @docid
+     * @public
+     */
+    filterBuilder?: dxFilterBuilderOptions;
```

Comment:
You can create a local public alias, so that it get into `card_view_types.d.ts` and aggregated exports in framework packages:

```ts
/**
 * @public
 */
type FilterBuilder = dxFilterBuilderOptions;
export {
  FilterBuilder
}
```

```suggestion
    filterBuilder?: FilterBuilder;
```

Reply:
Fixed

Question.
Will it be ok to:
```ts
export type FilterBuilder = dxFilterBuilderOptions;
```

Now I fixed exactly how u provided to exclude any risks (if I miss sth and there is a difference)